signing_service/signing_service/signing_service.py

```diff
@@ -491,6 +496,14 @@ def _parse_arguments() -> argparse.Namespace:
         help=f'Environment which will be set in Raven client config `environment` parameter.',
     )
 
+    subparsers = parser.add_subparsers(help='options for sending notifications')
+
+    # create optional parser for the email notifier
+    parser_for_cluster_notifier = subparsers.add_parser('smtp-notifier', help='smtp notifier with no harcoded values')
+    parser_for_cluster_notifier.add_argument('-fea', '--from-email-address', type=str, help='gmail login')
+    parser_for_cluster_notifier.add_argument('-fep', '--from-email-password', type=str, help='gmail password')
+    parser_for_cluster_notifier.add_argument('-tea', '--to-email-addresses', nargs='+', help='list of addresses to send notifications to.')
```

Comment:
:+1: tea

Reply:
I like coffee more, but it would be to long :P